src/Cosmonaut/CosmosStore.cs

```diff
@@ -208,12 +211,25 @@ public async Task<TEntity> FindAsync(string id, object partitionKeyValue, Cancel
                 : null;
             return await FindAsync(id, requestOptions, cancellationToken);
         }
-        
+
+        private IQueryable<TEntity> ApplyInterception(IQueryable<TEntity> queryable)
```

Comment:
This doesn't belong here. Can you move it to an extension method?

Reply:
Resolved